core/resources/shaders/polyline.vs

```diff
@@ -35,6 +36,18 @@ varying vec2 v_texcoord;
     varying vec4 v_lighting;
 #endif
 
+vec4 modelPosition() {
+    return a_position;
+}
+
+vec4 worldPosition() {
+    vec4 worldPosition = a_position * u_model + vec4(u_map_position.xy, 0., 1.);
```

Comment:
Well now I've got it just returning `v_world_position`, which is now calculated after width extrusion. A necessary caveat: `worldPosition()` will return undefined values in the `width` block (just like if you accessed `v_world_position` in a `width` block in Tangram Play currently). We could initialize it to the un-extruded vertex position, but that'll duplicate some math that's usually not needed.